pkg/oci/spec.go

```diff
@@ -173,9 +205,9 @@ func populateDefaultUnixSpec(ctx context.Context, s *Spec, id string) error {
 				GID: 0,
 			},
 			Capabilities: &specs.LinuxCapabilities{
```

Comment:
starting to wonder if we should move the capabilities out entirely, and instead have a `WithDefaultCapabilities`, `WithExtendedCapabilities` (or "reduced" if we want to do the reverse), or a `WithCapabilities(profileNe)` functional options.